devlib/module/cgroups.py

```diff
@@ -441,7 +441,7 @@ def run_into(self, cgroup, cmdline):
         :returns: Output of command.
         """
         cmd = self.run_into_cmd(cgroup, cmdline)
-        raw_output = self.target.execute(cmd)
+        raw_output = self.target.execute(cmd, as_root=self.target.is_rooted)
```

Comment:
What about exposing this via the API:
```python
def run_into(self, cgroup, cmdline, as_root=self.target.is_rooted):
```

Maybe there could be use-cases in which we want to test that a user cannot spawn tasks into a cgroup...

Reply:
I've updated this to allow overriding the behaviour.

Reply:
Should update the docstring to document the new parameter.

Reply:
Updated.